### PR TITLE
Add tests for commands modules: [context, network, performance]

### PR DESCRIPTION
**Pull request by vmaxim**

### CODE REVIEW

File: test/unit/commands/context-specs.js [hunk 1]
```diff
@@ -1,68 +1,268 @@
...
+    });
+    it('should ignore events if status is not STATE_STOPPED', async () => {
+      await driver.startChromedriverProxy('WEBVIEW_1');
+      await driver.chromedriver.emit(Chromedriver.EVENT_CHANGED,
+        {state: 'unhandled_state'});
+      driver.onChromedriverStop.notCalled.should.be.true;
+    });
+    it('should reconnect if session already exists', async () => {
+      stubbedChromedriver.hasWorkingWebview = sinon.stub().returns(true);
+      driver.sessionChromedrivers = {'WEBVIEW_1': stubbedChromedriver};
```
Comment (jlipps) [inline test/unit/commands/context-specs.js]: no need for quotes around `WEBVIEW_1`

File: test/unit/commands/context-specs.js [hunk 2]
```diff
@@ -1,68 +1,268 @@
...
+      await driver.startChromedriverProxy('WEBVIEW_1');
+      driver.chromedriver.restart.notCalled.should.be.true;
+      driver.chromedriver.should.be.equal(stubbedChromedriver);
+    });
+    it('should restart if chromedriver has not working web view', async () => {
+      stubbedChromedriver.hasWorkingWebview = sinon.stub().returns(false);
+      driver.sessionChromedrivers = {'WEBVIEW_1': stubbedChromedriver};
```
Comment (jlipps) [inline test/unit/commands/context-specs.js]: extra quotes

File: test/unit/commands/context-specs.js [hunk 3]
```diff
@@ -1,68 +1,268 @@
...
+      sinon.stub(driver, 'startUnexpectedShutdown');
+      driver.curContext = 'WEBVIEW_1';
+      await driver.onChromedriverStop('WEBVIEW_1');
+      let arg0 = driver.startUnexpectedShutdown.getCall(0).args[0];
+      arg0.should.be.an('error');
+      arg0.message.should.include('Chromedriver quit unexpectedly during session');
+    });
+    it('should delete session if chromedriver in non-active context', async () => {
+      driver.curContext = 'WEBVIEW_1';
+      driver.sessionChromedrivers = {'WEBVIEW_2': 'CHROMIUM'};
```
Comment (jlipps) [inline test/unit/commands/context-specs.js]: unnecessary quotes

**Comment by imurchie** [comment]
Thanks for these!

**Pull request merged**